cpp/src/cider/exec/nextgen/utils/ExprUtils.h

```diff
@@ -67,12 +67,13 @@ inline operators::ExprPtrVector collectColumnVars(const operators::ExprPtrVector
 inline bool isSubsetExprVector(const operators::ExprPtrVector& a,
                                const operators::ExprPtrVector& b) {
   if (a.size() <= b.size()) {
-    for (size_t i = 0; i < a.size(); ++i) {
-      if (a[i] != b[i]) {
-        return false;
+    size_t a_index = 0;
+    for (size_t b_index = 0; b_index < b.size(); ++b_index) {
```

Comment:
should check a_index in the loop, otherwise, core dump happened when a_index >= a.size()